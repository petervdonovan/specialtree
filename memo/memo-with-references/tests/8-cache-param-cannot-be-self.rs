use memo_with_references::memo;

struct Foo;

impl Foo {
    #[memo(self)]
    fn memome(&self) {}
}

fn main() {}
