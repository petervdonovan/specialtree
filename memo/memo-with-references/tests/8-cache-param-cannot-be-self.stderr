error: cache param cannot be self
 --> tests/8-cache-param-cannot-be-self.rs:6:12
  |
6 |     #[memo(self)]
  |            ^^^^
